contracts/RefunderFactory.sol

```diff
@@ -0,0 +1,43 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.4;
+
+import "./IRegistry.sol";
+import "./IRefunder.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Clones } from "@openzeppelin/contracts/proxy/Clones.sol";
+
+contract RefunderFactory is Ownable {
+
+    address _masterRefunder;
+    address _registry;
+
+    event CreateRefunder(address indexed owner, address indexed refunderAddress);
+
+    constructor(address masterRefunder_, address registry_) {
```

Comment:
Instead of enforcing the refunderFactory to work with only one `masterRefunder` we could pass the `masterRefunder` as argument to the `createRefunder` function